Library/Formula/kvazaar.rb

```diff
@@ -15,7 +17,9 @@ class Kvazaar < Formula
 
   def install
     system "make", "-C", "src"
-    bin.install "src/kvazaar"
+    bin.install_symlink "src/kvazaar"
```

Comment:
Yeah. What's likely happening here is that it's symlinking the binary from the buildpath into the Cellar, and the second the buildpath stops existing, the link breaks. Maybe you wanted to do something like this instead?

```ruby
if build.head?
  system "make", "-C", "src", "install"
else
  system "make", "-C", "src"
  bin.install "src/kvazaar"
end
```

?

Although, it is likely that `install` needs a prefix argument passed to it so it doesn't dump it into whatever prefix is hardcoded upstream.